prawns/governance/src/handlers/treasury.Proposed.event.ts

```diff
@@ -0,0 +1,54 @@
+import { EventHandlerContext, ExtrinsicHandlerContext } from '@subsquid/substrate-processor';
+import { decodeAddress, encodeAddress } from '../utils';
+import { SubstrateNetwork, SubstrateTreasuryProposal } from '../model';
+import { getOrCreateGovernanceAccount } from '../utils';
+import { getTreasuryProposedEvent} from "./typeGetters/getTreasuryProposedEvent";
+import { getTreasuryProposedSpendCall} from "./typeGetters/getTreasuryProposeSpendCall";
+
+export default (network: SubstrateNetwork) =>
+  async (ctx: EventHandlerContext) => {
+    if (!ctx.event || !ctx.event.extrinsic) {
+      return;
+    }
+    const blockNumber = BigInt(ctx.block.height);
+    const date = new Date(ctx.block.timestamp);
+    const rootAccount = decodeAddress(ctx.event.extrinsic.signer);
+    const event = getTreasuryProposedEvent(ctx, network);
+    const call = getTreasuryProposedSpendCall(<ExtrinsicHandlerContext>ctx, network);
+
+    const beneficiary = '0x' + Buffer.from(call.beneficiary).toString('hex');
```

Comment:
I think we should use the `decodeAddress` function here because it's an address on a specific network!